### PR TITLE
[New Entry} Added NumPy trace file

### PR DESCRIPTION
**Pull request by cigar-galaxy82**
### Issue Solved

#4480 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/numpy/concepts/built-in-functions/terms/trace/trace.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+Title: '.trace()'
+Description: 'Calculates the sum of the elements along the main diagonal of an array.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Arrays'
+  - 'Functions'
+  - 'Matrices'
+  - 'NumPy'
```
Comment (SaviDahegaonkar) [inline content/numpy/concepts/built-in-functions/terms/trace/trace.md:11]:
```suggestion
Tags:
  - 'Arrays'
  - 'Functions'
  - 'Matrices'
  - 'NumPy'
```
Include only those tags that are present in the [tags.md](documentation/tags.md) list and add one if it is not available in the list, Here `Functions` is not present in the tags.md list you can add it first in the tags.md file.

File: content/numpy/concepts/built-in-functions/terms/trace/trace.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+The **`.trace()`** function returns the sum of the elements along the main diagonal of an array. The array must have at least two dimensions. If the array used has more than two dimensions then axes need to be specified to determine the 2D sub-array whose traces are returned.
+
+## Syntax
+
+```pseudo
+numpy.trace(a, offset=0, axis1=0, axis2=1, out=None)
```
Comment (SaviDahegaonkar) [inline content/numpy/concepts/built-in-functions/terms/trace/trace.md:22]:
```suggestion
numpy.trace(a, offset=0, axis1=0, axis2=1, out=None)
```
Including a type parameter would make this syntax more accurate though it is not a necessary change.

**Review by SaviDahegaonkar — CHANGES_REQUESTED** [review]
Hey @cigar-galaxy82 ,  
The entry is nicely written, I have suggested some changes please make them asap.

Thanks,
Savi

File: content/numpy/concepts/built-in-functions/terms/trace/trace.md (hunk 1 above)
Comment (cigar-galaxy82) [reply content/numpy/concepts/built-in-functions/terms/trace/trace.md:11]: Hi @SaviDahegaonkar I believe the Functions tag is in the `tags.md`

**Comment by cigar-galaxy82** [comment]
Hi @SaviDahegaonkar I have made all the suggested changes

**Review by SaviDahegaonkar — APPROVED** [review]
Hey @cigar-galaxy82 ,
LGTM! , I hereby approve this PR.

**Pull request merged**